models/events.py

```diff
@@ -30,12 +29,10 @@ class Location(BaseModel):
 
 
 
-# incomplete
-# TODO: fix this name to be single event (doesnt make sense for plural)
-class Events(BaseModel):
+class Event(BaseModel):
```

Comment:
The naming refactor was needed but it left some crumbs (probably due to using vim and not an actual refactoring tool lol) around line 50;

`class registration_from(Events)` -> `class registration_form(Event)`

no biggie, but this is most likely why CircleCI wasn't passing (doing it's job i guess)

Reply:
Yeah this makes total sense